src/arrangerApi.js

```diff
@@ -0,0 +1,41 @@
+import { ARRANGER_API } from '@arranger/components/dist/utils/config';
+import urlJoin from 'url-join';
+import { addDownloadHttpHeaders } from '@arranger/components/dist/utils/download';
+
+/* File from @arranger node module - customized to pass additional params in fetch request */
+
+let alwaysSendHeaders = { 'Content-Type': 'application/json' };
+
+const api = ({ endpoint = '', body, headers }) => fetch(urlJoin(ARRANGER_API, endpoint), {
+  method: 'POST',
+  headers: { ...alwaysSendHeaders, ...headers },
+  body: JSON.stringify(body),
+  credentials: 'include',
+}).then(r => r.json());
+
+export const graphql = body => api({ endpoint: 'graphql', body });
+
+export const fetchExtendedMapping = ({ graphqlField, projectId }) =>
```

Comment:
I think we don't need to copy `fetchExtendedMapping`, `addHeaders` and `getAlwaysAddHeaders`here if we don't override them.